src/overmap.cpp

```diff
@@ -4046,6 +4131,14 @@ bool overmap::place_special_attempt( overmap_special_batch &enabled_specials,
             continue;
         }
 
+        //Biomes: check how many we have already created that have a matching flag.
+        //...if that amount is more than set for that region, then do not place.
+        for( std::pair<std::string, int> special_counts : settings.overmap_feature_flag.special_counts ) {
+            if( special.flags.count( special_counts.first ) >= special_counts.second ) {
```

Comment:
There are some other ways too, but something like that should do the thing:

```suggestion
            if( special.flags.count( special_counts.first ) >= static_cast<size_t>( special_counts.second ) ) {
```